### PR TITLE
F 2408 Change all Edit profile components to Ant

### PR DESCRIPTION
**Pull request by RamRamez**
Issue #2408 

1. Change all Edit profile components to Ant
2. Change it from class-based to hook-based
3. Fix some edit profile bugs
4. Make upload image reusable
5. Redesign edit profile

### CODE REVIEW

**Comment by aminlatifi** [comment]
Changing native currency doesn't need making transaction, since it's not saved in smart contracts

File: src/components/EditTraceCommons.jsx [hunk 1]
```diff
@@ -137,7 +137,8 @@ TraceDescription.defaultProps = {
   initialValue: '',
 };
 
-const TracePicture = ({ picture, setPicture, traceTitle, disabled }) => {
+const TracePicture = ({ picture, setPicture, traceTitle, disabled, aspectRatio }) => {
+  const _picture = picture.startsWith('https://ipfs.giveth.io') ? picture.slice(22) : picture;
```
[the remark this answers is not in the document]
Comment (aminlatifi) [reply src/components/EditTraceCommons.jsx:141]: TracePicture is not needed to be changed, or we can define only one component to use everywhere (Editing Community, Campaign, Trace and Picture).

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/components/EditTraceCommons.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (RamRamez) [reply src/components/EditTraceCommons.jsx:141]: I made a separate component for all image upload tasks

**Comment by RamRamez** [comment]
> Changing native currency doesn't need making transaction, since it's not part of user data in smart contracts

Done

**Pull request merged**